generators/app/templates/js/store/configureStore.js

```diff
@@ -6,13 +6,12 @@ let createStoreWithMiddleware;
 
 // Configure the dev tools when in DEV mode
 if (__DEV__) {
-  let {devTools, persistState} = require('redux-devtools');
+  const {devTools, persistState} = require('redux-devtools');
```

Comment:
It would be better to fix [this](https://github.com/banderson/generator-redux/blob/master/generators/app/templates/js/utils/devTools.js#L5) too, I think.
from `let` to `const`